resources/src/event_handlers/team_join_event_handler.rs

```diff
@@ -0,0 +1,15 @@
+use crate::commands;
+use serde_json::Value;
+pub async fn handle_team_join_event(body: &Value) {
+    // Deconstruct the body
+    let username: &str = body["event"]["user"]["id"]
+        .as_str()
+        .unwrap_or("invalid_user_name");
+    let first_name: &str = body["event"]["user"]["profile"]["first_name"]
+        .as_str()
+        .unwrap_or("");
```

Comment:
No error message?

Reply:
In this case, I left it as an empty string because it gets formatted to the message to use their name. I didn't want the message to read `Hey welcome to CodeDevils invalid_first_name` rather it just says currently `Hey welcome to CodeDevils` I could do a check before sending the message to match against `invalid_first_name` and to replace it with an empty string in the message so that it doesn't pop up to the user.